festflow/templates/404.html

```diff
@@ -0,0 +1,16 @@
+{% extends "festflow/base.html" %}
+{% load static %}
+{% block title %} Error 404 {% endblock %}
+
+{% block body_contents %}
+
+<div class="container">
+    <div class="title">
+        <div class="logo">
+            <img src="{% static "images/logo.png" %}" class="img-responsive" alt="Logo Brand">
+        </div>
+        <h1>404 ERROR</h1>
+        <h4>This page isn't available. Sorry about that.</h4>
```

Comment:
@akshatnitd  Please change this text to something like 'The page you are looking for does not exist.'